hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryManager.java

```diff
@@ -156,18 +148,6 @@ public void reviewExistingPayerRecords() {
 	 */
 	public void reviewExistingShortLivedEntities() {
```

Comment:
This is dead code now (c.f. now only test methods call `TransactionContext.addExpiringEntities()`).

Reply:
Not urgent, we could remove in a separate PR.